### PR TITLE
jooby: raw-jdbc, postgresql, updates/queries tests and latest release

### PR DESCRIPTION
**Pull request by jknack**

### CODE REVIEW

**Comment by jknack** [comment]
There were a couple of errors/failure, all them are fixed now for Jooby.

Thanks.

File: frameworks/Java/jooby/src/main/java/com/techempower/App.java [hunk 1]
```diff
@@ -44,57 +54,145 @@
...
-          .get()
-          .first();
-      return result(world, json);
+
+    get("/plaintext", (req, rsp) -> {
+      rsp.send(result(HELLO_WORLD_BUFFER, plain));
+    }).renderer("byteBuffer");
+
+    get("/json", (req, rsp) -> {
+      rsp.send(result(mapper.createObjectNode().put("message", HELLO_WORLD), json));
```
Comment (nbrady-techempower) [inline frameworks/Java/jooby/src/main/java/com/techempower/App.java]: Can we address #2888 here?

Comment (jknack) [inline frameworks/Java/jooby/src/main/java/com/techempower/App.java]: Sure, going to change. I copied from: https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/Java/play2-java/play2-java/app/controllers/Application.java

Comment (nbrady-techempower) [inline frameworks/Java/jooby/src/main/java/com/techempower/App.java]: @jknack Oh, thank you! Feel free to make the exact same change there if you'd like, and I'll add it to that issue for tracking.

**Comment by jknack** [comment]
Nate, done.

Thanks.

**Comment by nbrady-techempower** [comment]
Thank you!

**Pull request merged**